### ISSUE TITLE
Undock Tabs

### ISSUE THREAD

**Issue by Harinus** [zaproxy/zaproxy#2109]
Hi, 

a feature to undock Tabs for example the Script Console or the History would be nice!
Similar to the behaviour of eclipse.

Best Regards
Martin

**Comment by kingthorin** [comment]
Try double clicking a tab. Does that accomplish what you're after?

**Comment by Harinus** [comment]
No, double clicking a tab maximizes it. I want to have it in a seperate Window (for moving to to a second monitor for example) implies the ability to snap it back into the main window as a tab. IDEs like Eclipse and Visual Studio support what i am talking about. Would be nice to see the same thing in ZAP.

**Comment by kingthorin** [comment]
Oh ok, thanks for clarifying.
